code/game/objects/items/toys.dm

```diff
@@ -114,6 +114,14 @@
 	icon_state = "syndballoon"
 	item_state = "syndballoon"
 	w_class = WEIGHT_CLASS_BULKY
+	var/lastused = null
+
+/obj/item/toy/syndicateballoon/attack_self(mob/user)
+	if(world.time - lastused < CLICK_CD_MELEE)
+		return
+	var/playverb = pick("bat [src]", "tug on [src]'s string", "play with [src]")
+	user.visible_message("<span class = 'notice'>[user] plays with [src].</span>", "<span class = 'notice'>You [playverb].</span>")

```

Comment:
No spaces between `class = '`, please make it `class='`

Reply:
👍